ffs/rpc/rpc.go

```diff
@@ -728,8 +728,10 @@ func toRPCCidInfo(info ffs.CidInfo) *CidInfo {
 			},
 		},
 		Cold: &ColdInfo{
+			Enabled: info.Cold.Enabled,
```

Comment:
`ColdInfo` was missing a flag indicating if _ColdStorage_ is enabled (since you can store things in hot storage).
The `HotInfo` already had this flag, so now it's symmetric.

Reply:
Nice, I've noticed this and wished it existed.

Reply:
Something in my distant memory says the `ColdInfo.Validate()` can be improved by using this.

Reply:
Never mind, was thinking of `ColdConfig`

Reply:
Well, _ColdInfo_ doesn't have a _Validate_ method since it's only describing the current state of the storage. I think you're referring to _ColdConfig_, which already had the _Enabled_ flag and use it in `Validate()`. Very similar names. :P